example/index.jsx

```diff
@@ -91,13 +91,30 @@ const DatePickerSingle = React.createClass({
 
 
 var mainCodeSnippet = fs.readFileSync(__dirname + '/code-snippets/main.jsx', 'utf8');
+var i18nCodeSnippet = fs.readFileSync(__dirname + '/code-snippets/i18n.jsx', 'utf8');
 
 
 const Index = React.createClass({
+
+  getInitialState() {
+    return {
+      locale: 'en',
+    };
+  },
+
   getDefaultProps() {
     return {};
   },
 
+  _selectLocale() {
+    require(`moment/locale/${this.refs.locale.value}`);
```

Comment:
Do we need this line, if we've already loaded it as part of `example/code-snippets/i18n.jsx` ?